3]. Competitive Programming/LeetCode/Problems/Python/0002)Add_two_numbers.py

```diff
@@ -0,0 +1,42 @@
+# Definition for singly-linked list.
+# class ListNode:
+#     def __init__(self, val=0, next=None):
+#         self.val = val
+#         self.next = next
+
+
+class Solution:
+    
+    def addTwoNumbers(self, l1: ListNode, l2: ListNode) -> ListNode:
```

Comment:
change this line `def addTwoNumbers(self, l1: ListNode, l2: ListNode) -> ListNode:` to `def addTwoNumbers(self, l1, l2):`

Because this generates `Runtime Error`